src/Shared/Helpers/SarifBuilder.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.CodeAnalysis.Sarif;
+using NLog;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Text;
+
+namespace Microsoft.CST.OpenSource.Shared
+{
+    public class SarifBuilder
+    {
+        /// <summary>
+        /// Class logger
+        /// </summary>
+        protected static readonly NLog.Logger Logger = NLog.LogManager.GetCurrentClassLogger();
+        string sarifVersion = "sarifv2";
```

Comment:
If this is a constant, SARIF_VERSION_STRING or something, and private/readonly.

Reply:
its a private class member which is assigned in the constructor - I can make it readonly though